doc/markup/inline.rst

```diff
@@ -154,6 +154,21 @@ Cross-referencing arbitrary locations
    section headings are changed, and for all builders that support
    cross-references.
 
+   .. versionadded:: 1.4
```

Comment:
Judging by the other examples in the doc, and having looked at the output, this doesn't appear to be the case.
I've incorporated your other suggestions; thanks for the feedback!